Cloud Application Manager/Tutorials/ebcli-tutorial.md

```diff
@@ -0,0 +1,442 @@
+{{{
+"title": "Using ebcli",
+"date": "05-20-2019",
+"author": "Diego Sanjuan",
+"attachments": [],
+"contentIsHTML": false
```

Comment:
Please, you should also include `keywords` here

Reply:
done